local/rpm/Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: run
+run:
+	@podman run -ti -p 3128:3128 -d --name forwarder-rpm forwarder-rpm
+	@podman exec -ti forwarder-rpm /bin/sh
+
+.PHONY: build
+build: package
+	@cp ../../dist/forwarder*_linux.aarch64.rpm .
+	@mv forwarder*_linux.aarch64.rpm forwarder.aarch64.rpm
+	@podman build -t forwarder-rpm .
+	@rm forwarder.aarch64.rpm
+
+.PHONY: package
+package:
+	@cd ../../ && ./bin/goreleaser release --skip-publish --snapshot --skip-docker --clean
```

Comment:
This can build only linux images.

Reply:
How do I build nfpms only?

Reply:
How about --single-target?

Reply:
This works with build only... https://github.com/goreleaser/goreleaser/issues/3219